scss/_variables.scss

```diff
@@ -1696,6 +1696,9 @@ $modal-header-padding-y:            0 !default;
 $modal-header-padding-x:            $spacer !default;
 $modal-header-padding:              $modal-header-padding-y $modal-header-padding-x !default; // Keep this for backwards compatibility
 
+$modal-body-margin-bottom:          $spacer * .5 !default; // Boosted mod
+$modal-body-margin-sm-bottom:       $spacer * .75 !default; // Boosted mod
```

Comment:
If I'm not mistaken, most of our Sass vars have `-sm` as a suffix. Shouldn't it be `$modal-body-margin-bottom-sm` here?

Reply:
Not always. We also have `$modal-dialog-margin-y-sm-up` or `$modal-content-box-shadow-sm-up` for instance.
It seems we need to have `-<breakpoint>-<position>`

Reply:
OK so let's go as is. @louismaximepiton OK for you?

Reply:
Looking at this, I feel like the `sm-up` stands for breakpoint and above. I think that it should definitely be `$modal-body-margin-bottom-sm`